test_unstructured/cleaners/test_extract.py

```diff
@@ -21,3 +24,25 @@ def test_extract_text_before():
 def test_extract_text_after():
     text = "Teacher: BLAH BLAH BLAH; Student: BLAH BLAH BLAH!"
     assert extract.extract_text_after(text, "BLAH;", 0) == "Student: BLAH BLAH BLAH!"
+
+
+def test_extract_email_address():
+    text = "Imran Scotty <Imran.Scotty@npf.gov.nr>"
+    assert extract.extract_email_address(text) == ["imran.scotty@npf.gov.nr"]
+
+
+def test_extract_ip_address():
+    assert extract.extract_ip_address(EMAIL_META_DATA_INPUT) == ["ba23::58b5:2236:45g2:88h2",
+     "ba23::58b5:2236:45g2:88h2%25"]
+
+
+def test_extract_ip_address_name():
+    assert extract.extract_ip_address_name(EMAIL_META_DATA_INPUT) == ["ABC.DEF.local","ABC.DEF.local"]
+
+
+def extract_mapi_id():
+    assert extract.extract_ip_address_name(EMAIL_META_DATA_INPUT) == ["32.88.5467.123"]
```

Comment:
Should all of these be `extract_ip_address_name` or should they be `extract_datetimetz`, `extract_mapi_id`, etc?

Reply:
Oops! Thanks for catching that!